### DISCUSSION TITLE
max_over_time(sum

### DISCUSSION THREAD

**Discussion by reyes136** [prometheus/prometheus#10929] [Q&A]
I am trying to get this to run but it doesn't seem to work. I have read some examples where it says it should work.

max_over_time(sum(pdu_inlet_sensor_value{instance=~"10.241.137.132|10.241.137.134|10.241.137.135|10.241.137.137",sensorType="apparentPower"})[15d])

This query bellow does work but it returns the wrong value, seems to take the max of all the instances then the max of them over time which returns too large a value.

sum(max_over_time(pdu_inlet_sensor_value{instance=~"10.241.137.132|10.241.137.134|10.241.137.135|10.241.137.137",sensorType="apparentPower"}[5d]))

**Comment by roidelapluie** [comment]
What are you trying to achieve?

**Comment by reyes136** [comment]
I am taking the power reading from a PDU every 3 min for around 60 different PDU's. I need to get the max power used by all PDU's during the last 15 days
